src/Controls/src/Core/MessagingCenter.cs

```diff
@@ -202,21 +202,20 @@ void InnerSend(string message, Type senderType, Type argType, object sender, obj
 			if (message == null)
 				throw new ArgumentNullException(nameof(message));
 			var key = new Sender(message, senderType, argType);
-			if (!_subscriptions.ContainsKey(key))
+			if (!_subscriptions.TryGetValue(key, out List<Subscription> value))
```

Comment:
use a better name, like subscriptions

Reply:
subscriptions would seem to conflict with `_subscriptions`.